packages/code-studio/src/settings/FormattingSectionContent.tsx

```diff
@@ -596,6 +613,23 @@ export class FormattingSectionContent extends PureComponent<
               </Checkbox>
             </div>
           </div>
+
+          <div className="form-row mb-3">
+            <label
+              className="col-form-label col-3"
+              htmlFor="default-integer-format-input"
+            >
+              Rollup
+            </label>
+            <div className="col pr-0 pt-2">
+              <Checkbox
+                checked={showExtraGroupColumn ?? null}
```

Comment:
Shouldn't need the `?? null` here. This value should have a default value as part of the default workspace and never be null or undefined here